### ISSUE TITLE
Unable to change font family for Body and Heading

### ISSUE THREAD

**Issue by alex-hsa** [forumone/gesso-uswds#283]
Hello:

After changing the fallback options in the 'config.design-tokens.yml file our custom fonts don't get displayed.

What are the proper steps to change the fonts for Body and Heading fonts?

Any insight would be highly appreciated. Thank you very much.

**Comment by coreylafferty** [comment]
@kmonahan @dcmouyard I looked into this a bit. I haven't found the root cause, but can confirm the font families called in the 'display' section of the design tokens don't get rendered in the CSS. 

```
h1:
        color: brand.blue.dark-1
        font-family: heading.stack
```

While color gets created in the CSS, font-family isn't there at all.

`heading.stack` is coming from here:

```
heading:
        name: Heading
        stack: !sass
          sass: 'font-family(heading)'
          fallback: '"Merriweather Web", Georgia, Cambria, "Times New Roman", Times, serif'
```

I think this has something to do with the `!sass` bit, because if I make it this:

```
heading:
        name: Heading
        stack: 'font-family(heading)'
```

it does create the font-family property in the generated CSS file.

**Comment by coreylafferty** [comment]
Fixed by #361
